src/routes/CommunitySidebar/CommunitySidebar.js

```diff
@@ -24,6 +24,8 @@ export default class CommunitySidebar extends Component {
   render () {
     const { community, members, leaders, canModerate } = this.props
 
+    if (community.memberCount === 0) return null
```

Comment:
why is this needed? it should certainly be below the next conditional which checks if community exists.

Reply:
This is meant to hide the sidebar when a person views a community they do not have access to... But I think that might be an edge case possibly coming from my local data - I was once a member of a community and left, and so I can access the `c/slug` url. Whereas if I was never a member of a community, it shows the 404 page.